tomster-player/app/adapters/application.js

```diff
@@ -5,16 +5,18 @@ import ENV from 'tomster-player/config/environment';
 
 export default class ApplicationAdapter extends JSONAPIAdapter {
   @service session;
+  @service socket;
 
   host = ENV.apiHost;
   namespace = ENV.apiNamespace;
 
-  @computed('session.{isAuthenticated,data.authenticated.access_token}')
+  @computed('session.{isAuthenticated,data.authenticated.access_token}', 'socket.id')
```

Comment:
Do we still need to mention the `socket.id` key within the computed decorator when it is marked as `tracked` within the socket service?

Reply:
Yes, I think so 😞